package back.auth.review.common.exception;

public class AuthException extends RuntimeException{
}
